backend/src/application/__init__.py

```diff
@@ -48,4 +49,6 @@ def create_app(config):
     admin.add_view(UserAdminView(User, db.session))
     admin.add_view(ModelView(Group, db.session))
 
+    CORS(app, resources={r"/api/*": {"origins": "*"}})
```

Comment:
Uhm in the end we should have here just the domain the page will be hosted on, right?

Maybe move the origins value to config, so that `*` will be only for a dev config?